concierge/service/src/main/java/org/eclipse/ditto/concierge/service/enforcement/PolicyCommandEnforcement.java

```diff
@@ -367,6 +385,7 @@ private static ResourceKey asResourceKey(final Label label, final PolicyCommand<
 
         private final Cache<EnforcementCacheKey, Entry<PolicyEnforcer>> enforcerCache;
         private final ActorRef policiesShardRegion;
+        private final CreationRestrictionEnforcer creationRestrictionEnforcer;
```

Comment:
Please mark the field as `@Nullable`, the constructor param also is. Or do the fallback to the `CreationRestrictionEnforcer.NULL` already in the constructor in order to avoid the `nullable` field